packages/api-reference/test/utils/sources.ts

```diff
@@ -12,19 +13,19 @@ export const sources = [
   {
     title: 'Scalar Galaxy', // optional, would fallback to 'API #1'
     slug: 'scalar-galaxy', // optional, would be auto-generated from the title or the index
-    url: 'https://galaxy.scalar.com/openapi.yaml',
-  },
-  {
-    title: 'Scalar Galaxy Registry',
-    slug: 'scalar-galaxy-registry',
-    url: 'https://registry.scalar.com/@scalar/apis/galaxy?format=json',
+    content: galaxy,
   },
   {
     title: 'Scalar Galaxy (Classic Layout)',
     slug: 'scalar-galaxy-classic',
-    url: 'https://registry.scalar.com/@scalar/apis/galaxy?format=json',
+    content: galaxy,
```

Comment:
If we'd prefer to use this over a URL we can have `serveExample` host it instead